### PR TITLE
ability to set priority attribute

### PR DESCRIPTION
**Pull request by gardenia**
added ability to set priority attribute

references:
https://developers.google.com/cloud-messaging/http-server-ref
https://developers.google.com/cloud-messaging/concept-options?hl=en#setting-the-priority-of-a-message

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,4 +13,5 @@
     keywords='android gcm push notification google cloud messaging',
     install_requires=['requests'],
     tests_require=['mock'],
+    test_suite='gcm.test',
```
Comment (mnemonicflow) [inline setup.py]: This is useful, but can you please add this change to another commit?

**Comment by mnemonicflow** [comment]
Thanks for the pull request. Can you please remove the link from the commit message and add the test_suit change in another commit?

**Comment by gardenia** [comment]
PR updated as requested

**Comment by mnemonicflow** [comment]
Looks like you screwed up the commits. I've grabbed your fork and did the following things to correct the problems:
1) git rebase -i -p a418d87a670b6a167cdc0548752282456dcdc9e9
2) delete the following two commits "Merge branch 'master' of https://github.com/gardenia/python-gcm" and " added ability to set priority (https://developers.google.com/cloud-me"
(make sure to save and quit the editor for the rebase operation to continue; you can also set the editor used by rebase by specifying the GIT_EDITOR enviroment variable)
3) git log
Make sure you only see the commits with the messages 'added ability to set priority' and 'added test_suite attribute to setup.py (facilitates tests being run via "python setup.py test")'
3) git remote add upstream git@github.com:geeknam/python-gcm.git
4) git pull --rebase upstream master
4) git push --force origin master

**Comment by gardenia** [comment]
done

**Comment by mnemonicflow** [comment]
Looks Good To Me! :+1:

**Pull request merged**